packages/web-pkg/src/components/AppTopBar.vue

```diff
@@ -184,7 +184,11 @@ export default defineComponent({
 
 .open-file-bar {
   #app-top-bar-resource {
-    max-width: 50vw;
+    max-width: 360px;
+
+    @media (max-width: $oc-breakpoint-small-default) {
```

Comment:
Please use `$oc-breakpoint-medium-default`. Otherwise you'll have this:

<img width="660" alt="Screenshot 2023-11-27 at 09 42 04" src="https://github.com/owncloud/web/assets/3532843/bc229d4a-3e4f-4595-ac6d-a385f7ceafb8">